src/code/fv-components/egg-hatch-drake.js

```diff
@@ -0,0 +1,53 @@
+import React, { PropTypes } from 'react';
+import AnimatedSprite from '../components/animated-sprite';
+import OrganismView from '../components/organism';
+import classNames from 'classnames';
+
+export default class EggHatchDrakeView extends React.Component {
+
+  static propTypes = {
+    id: PropTypes.string,
+    eggClasses: PropTypes.string,
+    drakeClasses: PropTypes.string,
+    drake: PropTypes.object,
+    hatchStarted: PropTypes.bool,
+    hatchComplete: PropTypes.bool,
+    onHatchComplete: PropTypes.func
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
`hatchComplete` indicates whether the drake is hatched. That state can be achieved either by showing the animation and detecting when it has completed, or by the client stating that the drake is already hatched, in which case the animation is not required. I'm still inclined to leave things as they are.